src/dbos.ts

```diff
@@ -224,12 +225,27 @@ export class DBOS {
     return DBOSExecutor.globalInstance as DBOSExecutorContext;
   }
 
+  static async launchAppHTTPServer() {
```

Comment:
Do users need to manually invoke this in their app, or is it just for testing?

Reply:
This is the current solution for testing purposes.
The HTTP server had been held by testingRuntime, which is not in use.
launch() had been making it under certain conditions; @maxdml changed those.  We may clean up a bit after the dust settles there.